src/layouts/MainHeader/MainHeader.tsx

```diff
@@ -1,21 +1,15 @@
 import { FC } from "react";
-import { Search } from "@mui/icons-material";
 
 import { GroupMenu } from "@components/GroupMenu/GroupMenu.tsx";
 
 import "./MainHeader.css";
 
+import { SearchBar } from "@/components/Header/SearchBar/SearchBar.tsx";
```

Comment:
```suggestion
import { SearchBar } from "@components/Header/SearchBar/SearchBar.tsx";
```
**Note** : pas la bonne place import